src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -51,11 +51,14 @@
 #include <exception>
 #include <string>
 #include <thread>
+#include <vector>
 
 #ifndef NO_TTF
 #    include "../drawing/TTF.h"
 #endif
 
+using argv_t = std::vector<std::basic_string<utf8>>;
```

Comment:
std::basic_string<utf8> is currently identical to std::string. I'm not sure why we have a typedef utf8 which is essentially char right now. std::string should be able to carry UTF8 generally without a special storage type. So you might as well just use the plain std::string type.

Reply:
So, don't we want to get rid of the utf8 typedef at all then?

Reply:
We do, eventually everything will be `std::string` or `std::string_view`.

Reply:
Good to know!